plugins/GSdx/GSRendererHW.cpp

```diff
@@ -153,7 +153,9 @@ bool GSRendererHW::CanUpscale()
 		return false;
 	}
 
-	return m_upscale_multiplier!=1 && m_regs->PMODE.EN != 0; // upscale ratio depends on the display size, with no output it may not be set correctly (ps2 logo to game transition)
+	// Upscale ratio depends on the display size, with no output it may not be set correctly (ps2 logo to game transition)
+	// Only allow upscaling when either one of the output circuit is enabled.
+	return m_upscale_multiplier!=1 && m_regs->PMODE.EN != 0 && (m_regs->PMODE.EN1 || m_regs->PMODE.EN2);
```

Comment:
Just found the existence of the `EN` field. So it is a 2 bits concatenation of EN1 and EN2. So if either EN1 or EN2 is 1 then EN != 0. And if both EN1 and EN2 == 0 then EN == 0.